### ISSUE TITLE
Dropdown Component has no accessibility - NVDA

### ISSUE THREAD

**Issue by capivaramu** [primefaces/primeng#9342]
I'm submitting a ...

[ x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[x ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35

**Current behavior**
When using arrow navigation, the dropdown does not display all options.
When using the NVDA accessibility program, he cannot read the internal content of the dropdown, always informing the message "blank"

**Expected behavior**
I would like to request that the component has accessibility for navigation with TAB / keyboard arrows, and that programs like NVDA start to work, we have a growing community in terms of accessibility, so I think it would be very fair to help people with visual impairments

**Minimal reproduction of the problem with instructions**
The test was performed using the arrows, and the dropdown does not change the option with the arrows even without opening the options just to have the focus.
NVDA cannot read options

**What is the motivation / use case for changing the behavior?**
Adaptation for sites that incorporate accessibility in new projects use primeng as a reference in this area.

**Comment by yigitfindikli** [comment]
Fixed in https://github.com/primefaces/primeng/issues/9834

**Comment by morane1** [comment]
@yigitfindikli 

  still not working on PrimeNg 12.2.1 

When using the NVDA accessibility program, he cannot read the internal content of the dropdown, when you navigate with arrows it only informing on selected item.
